dces-drc-integration/src/integrationTest/java/uk/gov/justice/laa/crime/dces/integration/service/spy/ContributionProcessSpy.java

```diff
@@ -62,7 +62,7 @@ public ContributionProcessSpyBuilder traceGetContributionsActive() {
                 @SuppressWarnings("unchecked") final var result = (List<ConcorContribEntry>) mockingDetails(contributionClientSpy).getMockCreationSettings().getDefaultAnswer().answer(invocation);
                 activeIds(result.stream().map(ConcorContribEntry::getConcorContributionId).collect(Collectors.toSet()));
                 return result;
-            }).when(contributionClientSpy).getContributions("ACTIVE", 0, 10);
+            }).when(contributionClientSpy).getContributions(eq("ACTIVE"), eq(0), anyInt());
```

Comment:
It could yeah, but would mean that we'd never consume any real data.
It was a toss up between real dev data, or infinite loops. If it hits a second page.